examples/mobile/UIComponents/components/dialogs/index.html

```diff
@@ -13,44 +13,19 @@
 	<div class="ui-page" id="popup-demo-page">
 		<div class="ui-header" data-position="fixed">
 			<h1>
-				Popup
+				Dialogs
 			</h1>
 		</div>
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
 				<li class="ui-li-anchor">
 					<a href="popup.html">
-						Popup
+						General Dialogs
```

Comment:
Check whether it is General Dialog / Dialog.

Reply:
I make a "General Dialogs" for distinguish with Picker Dialog.
If it confuses you, I'll change "Dialog".

Reply:
ok. using General Dialogs. agree.